.github/workflows/new-app-review/README.md

```diff
@@ -0,0 +1,26 @@
+# New App Review
+
+PRs containing new app submissions undergo some extra validation to ensure they are ready for review.
+
+This workflow script will run these checks, comment on the PR with any failures,
+and label the PR according to the review result as either `Ready For Review` or `Needs Work`.
```

Comment:
How about: 
"Pending Contentful Review"
"Partner Action Needed"
?

Just to be more partner language friendly and explicitly include the owner of the review status. That is, assuming these are external facing terms.

Reply:
Thanks for the feedback! Updated the labels 😄